library/panic_unwind/Cargo.toml

```diff
@@ -16,7 +16,7 @@ alloc = { path = "../alloc" }
 core = { path = "../core" }
 unwind = { path = "../unwind" }
 compiler_builtins = "0.1.0"
-cfg-if = "1.0"
+cfg-if = { version = "1.0", features = ['rustc-dep-of-std'] }
```

Comment:
what is this change for?

Reply:
When building with `x test test/mir-opt`, I noticed that std keeps rebuilding after no changes in it.
Setting `CARGO_LOG=cargo::core::compiler::fingerprint=info`, it reveals that cfg-if built different
for panic_unwind and std.
Having said that, I try again with simple `x build libary` multiple times without this change,
std is not getting rebuilt when unchanged.

Reply:
I'm happy to remove this diff if it adds noice.

Reply:
That's a good find, we should keep this change.